castle/cms/profiles/2_2_0/browserlayer.xml

```diff
@@ -0,0 +1,6 @@
+<layers>
+  <layer
+    name="castle.cms.pwexpiry"
+    interface="castle.cms.pwexpiry.interfaces.ICollectivePWExpiryLayer"
+    remote="True" />
```

Comment:
typo, `remove`, not `remote`

Reply:
Just following the [directions given](https://github.com/castlecms/castle.cms/pull/243#discussion_r233197514) 😂